pkg/agent/secondarynetwork/podwatch/types.go

```diff
@@ -38,5 +39,7 @@ type SecondaryNetworkConfig struct {
 	Type string `json:"type,omitempty"`
 	// Set networkType to "sriov"
 	NetworkType string     `json:"networkType,omitempty"`
+	MTU         int        `json:"mtu, omitempty"`
+	VLAN        uint16     `json:"vlan, omitempty"`
```

Comment:
probably better to use `int32` in a public API, especially since the max value is 4096 and you have to perform validation anyway
even though this is not a K8s API: https://github.com/zecke/Kubernetes/blob/master/docs/devel/api-conventions.md#primitive-types

Reply:
Changed to int32